VKClientRM/VKClientRM/Model/newsfeedGet/ResponseVKNews.swift

```diff
@@ -0,0 +1,16 @@
+// ResponseVKNews.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Ответ с сервера о новости вконтакте
+final class ResponseVKNews: Decodable {
+    /// Новости
+    let items: [Newsfeed]
+
+    // MARK: - CodingKeys
+
+    enum CodingKeys: String, CodingKey {
```

Comment:
```suggestion
    enum CodingKeys: String, CodingKey {
```
А без него будет работать ?